### PR TITLE
systemd: Document InfluxDB data storage location

### PR DESCRIPTION
**Pull request by pierwill**
Work toward https://github.com/influxdata/docs-v2/issues/2136.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/get-started.md [hunk 1]
```diff
@@ -209,6 +209,10 @@ If you rename the binaries, all references to `influx` and `influxd` in this doc
 
 ### Install InfluxDB as a service with systemd
 
+{{% note %}}
+As of InfluxDB 2.0.3, `systemd` is required for Linux service support.
```
Comment (pierwill) [inline content/influxdb/v2.0/get-started.md:213]: This from @psteinbachs.

Comment (psteinbachs) [reply content/influxdb/v2.0/get-started.md:213]: I had @danxmoran double check this, and we still fall back to init in the deb/rpm packages.

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:213]: Ah! Okay. So let's delete this note, yes?

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:213]: Or edit it?

Comment (psteinbachs) [reply content/influxdb/v2.0/get-started.md:213]: I would just delete it for now, thanks.

File: content/influxdb/v2.0/get-started.md [hunk 2]
```diff
@@ -235,6 +239,8 @@ If you rename the binaries, all references to `influx` and `influxd` in this doc
      Active: active (running)
    ```
 
+When installed as a service, InfluxDB stores data in `/var/lib/influxdb/`.
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:242]: Is it just data or all InfluxDB files/directories (`configs/`, `engine/`, `influxd.bolt`)?

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:242]: Looks like configs are in `$HOME/.influxdbv2/` when installed on Ubuntu using `.deb`.

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:242]: `engine/` and `influx.bolt` are both in `/var/lib/influxdb`.

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:242]: Editing the PR now...

File: content/influxdb/v2.0/get-started.md [hunk 3]
```diff
@@ -235,6 +235,10 @@ If you rename the binaries, all references to `influx` and `influxd` in this doc
...
+(This includes `engine/` and `influxd.bolt`.)
+[`influx config`](/influxdb/v2.0/reference/cli/influx/config/) files are stored in `~/.influxdbv2/`.
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:240]:
```suggestion
When installed as a service, InfluxDB stores data in the following locations:

- **Time series data:** `/var/lib/influxdb/engine/`
- **Key-value data:** `/var/lib/influxdb/influxd.bolt`.
- **influx CLI configurations:** `~/.influxdbv2/configs` _(see [`influx config`](/influxdb/v2.0/reference/cli/influx/config/) for more information)_ .
```

Comment (pierwill) [reply content/influxdb/v2.0/get-started.md]: 👍

**Review by sanderson — APPROVED** [review]

**Pull request merged**